extensions/adobe/experience/analytics/keyedlist.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/keyedlist",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Keyed List",
+  "type": "object",
+  "description": "List of keyed values.",
+  "definitions": {
+    "keyedlist": {
```

Comment:
For other reviewers the reason we are defining this type vs just using an object with additionalProperties (a map) is for our usage in Analytics list variables the key is optional and we are also keeping open the option of adding the original delimiters to this struct later like we have for listdetails - https://github.com/adobe/xdm/blob/master/extensions/adobe/experience/analytics/listdetails.schema.json#L24

Reply:
I don't think it's the most elegant way to express something, but fine for an extension.